crates/adapters/dydx/src/execution/submitter.rs

```diff
@@ -280,87 +395,161 @@ impl OrderSubmitter {
         ))
     }
 
-    #[allow(dead_code)]
-    fn extract_market_params(
+    /// Convert Nautilus OrderSide to proto OrderSide.
+    fn convert_order_side(&self, side: OrderSide) -> ProtoOrderSide {
+        match side {
+            OrderSide::Buy => ProtoOrderSide::Buy,
+            OrderSide::Sell => ProtoOrderSide::Sell,
+            _ => ProtoOrderSide::Unspecified, // Should never happen after validation
+        }
+    }
+
+    /// Convert Nautilus TimeInForce to proto TimeInForce.
+    fn convert_time_in_force(&self, tif: TimeInForce) -> ProtoTimeInForce {
```

Comment:
This should be a function under `common/parse.rs` probably. Or, if a 1-to-1 mapping could implement `From` on the enum.